app/views/content/train-to-be-a-teacher/assessment-only-route-to-qts.md

```diff
@@ -13,7 +13,7 @@ fullwidth: true
 content:
   - content/train-to-be-a-teacher/assessment-only-route-to-qts/listing
   - content/train-to-be-a-teacher/assessment-only-route-to-qts/update-details
-
+  - content/shared/block-promos/mailing_adviser_routes
```

Comment:
I don't think it is appropriate to put mailing list here - AO candidates won't get any useful guidance from that

I also think you need to check with the candidate support team before adding advisers to this page - although I know it is in prod already!